samples/ios/app/glean-sample-app/Info.plist

```diff
@@ -2,6 +2,17 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
```

Comment:
So this becomes something that each app needs to implement, right? (which makes sense, because each app needs an individual scheme).

Reply:
Yes, it's looking like this will have to be implemented at the application level as it's not possible to receive the ApplicationDelegate from the Glean framework/library.  But this is all just experiments yet, I wasn't ready for further review yet so I haven't had time to test out what all is possible yet.